crates/transaction-emitter-lib/src/emitter/account_minter.rs

```diff
@@ -279,15 +285,34 @@ impl<'t> AccountMinter<'t> {
                     )
                 })
                 .collect();
-            execute_and_wait_transactions(
+            let result = execute_and_wait_transactions(
                 &client,
                 source_account,
                 create_requests,
                 failed_requests,
             )
-            .await?;
-            i += batch_size;
-            seed_accounts.append(&mut batch);
+            .await;
+
+            if let Err(err) = result {
+                warn!("Creating seed accounts batch failed {:?}, retrying", err);
+                loop {
```

Comment:
Not sure if I understand this -  will this not as simple as wrapping `execute_and_wait_transactions`  in a retry policy? The loop over here might get stuck for a very long time  - we probably need to do it with some timeout.

Reply:
we need to recreate a transaction (because of expiration time), and update sequence number appropriately (if some went through, but some didn't)

we could alternatively be setting large expiration, and be submitting transaction if it doesn't get committed every shorter amount of time

Reply:
Longer expiration time with retries sounds like a cleaner approach to me. Thoughts?